[2023-03-16T00:23:37.767+0000] {processor.py:153} INFO - Started process (PID=1979) to work on /opt/airflow/dags/nhe.py
[2023-03-16T00:23:37.769+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nhe.py for tasks to queue
[2023-03-16T00:23:37.771+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:23:37.770+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/nhe.py
[2023-03-16T00:23:37.780+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:23:37.776+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/nhe.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nhe.py", line 37
    if(qtd > 1000):
                  ^
TabError: inconsistent use of tabs and spaces in indentation
[2023-03-16T00:23:37.781+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nhe.py
[2023-03-16T00:23:37.810+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nhe.py took 0.050 seconds
[2023-03-16T00:24:08.276+0000] {processor.py:153} INFO - Started process (PID=2029) to work on /opt/airflow/dags/nhe.py
[2023-03-16T00:24:08.277+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nhe.py for tasks to queue
[2023-03-16T00:24:08.278+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:24:08.278+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/nhe.py
[2023-03-16T00:24:08.284+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:24:08.282+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/nhe.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nhe.py", line 37
    if(qtd > 1000):
                  ^
TabError: inconsistent use of tabs and spaces in indentation
[2023-03-16T00:24:08.286+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nhe.py
[2023-03-16T00:24:08.319+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nhe.py took 0.050 seconds
[2023-03-16T00:24:39.052+0000] {processor.py:153} INFO - Started process (PID=2079) to work on /opt/airflow/dags/nhe.py
[2023-03-16T00:24:39.055+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nhe.py for tasks to queue
[2023-03-16T00:24:39.056+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:24:39.056+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/nhe.py
[2023-03-16T00:24:39.065+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:24:39.063+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/nhe.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nhe.py", line 37
    if(qtd > 1000):
                  ^
TabError: inconsistent use of tabs and spaces in indentation
[2023-03-16T00:24:39.066+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nhe.py
[2023-03-16T00:24:39.090+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nhe.py took 0.044 seconds
[2023-03-16T00:25:09.700+0000] {processor.py:153} INFO - Started process (PID=2129) to work on /opt/airflow/dags/nhe.py
[2023-03-16T00:25:09.702+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nhe.py for tasks to queue
[2023-03-16T00:25:09.704+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:25:09.704+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/nhe.py
[2023-03-16T00:25:09.718+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:25:09.717+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/nhe.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nhe.py", line 37
    if(qtd > 1000):
                  ^
TabError: inconsistent use of tabs and spaces in indentation
[2023-03-16T00:25:09.719+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nhe.py
[2023-03-16T00:25:09.740+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nhe.py took 0.049 seconds
[2023-03-16T00:25:40.419+0000] {processor.py:153} INFO - Started process (PID=2179) to work on /opt/airflow/dags/nhe.py
[2023-03-16T00:25:40.422+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nhe.py for tasks to queue
[2023-03-16T00:25:40.424+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:25:40.424+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/nhe.py
[2023-03-16T00:25:40.431+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:25:40.429+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/nhe.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nhe.py", line 37
    if(qtd > 1000):
                  ^
TabError: inconsistent use of tabs and spaces in indentation
[2023-03-16T00:25:40.432+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nhe.py
[2023-03-16T00:25:40.455+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nhe.py took 0.042 seconds
[2023-03-16T00:26:11.319+0000] {processor.py:153} INFO - Started process (PID=2229) to work on /opt/airflow/dags/nhe.py
[2023-03-16T00:26:11.321+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nhe.py for tasks to queue
[2023-03-16T00:26:11.322+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:26:11.322+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/nhe.py
[2023-03-16T00:26:11.330+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:26:11.329+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/nhe.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nhe.py", line 37
    if(qtd > 1000):
                  ^
TabError: inconsistent use of tabs and spaces in indentation
[2023-03-16T00:26:11.331+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nhe.py
[2023-03-16T00:26:11.359+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nhe.py took 0.048 seconds
[2023-03-16T00:26:41.858+0000] {processor.py:153} INFO - Started process (PID=2279) to work on /opt/airflow/dags/nhe.py
[2023-03-16T00:26:41.861+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nhe.py for tasks to queue
[2023-03-16T00:26:41.863+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:26:41.863+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/nhe.py
[2023-03-16T00:26:41.872+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:26:41.870+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/nhe.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nhe.py", line 37
    if(qtd > 1000):
                  ^
TabError: inconsistent use of tabs and spaces in indentation
[2023-03-16T00:26:41.873+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nhe.py
[2023-03-16T00:26:41.898+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nhe.py took 0.045 seconds
[2023-03-16T00:27:12.583+0000] {processor.py:153} INFO - Started process (PID=2329) to work on /opt/airflow/dags/nhe.py
[2023-03-16T00:27:12.586+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nhe.py for tasks to queue
[2023-03-16T00:27:12.588+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:27:12.588+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/nhe.py
[2023-03-16T00:27:12.597+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:27:12.595+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/nhe.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nhe.py", line 37
    if(qtd > 1000):
                  ^
TabError: inconsistent use of tabs and spaces in indentation
[2023-03-16T00:27:12.598+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nhe.py
[2023-03-16T00:27:12.623+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nhe.py took 0.047 seconds
[2023-03-16T00:27:43.244+0000] {processor.py:153} INFO - Started process (PID=2379) to work on /opt/airflow/dags/nhe.py
[2023-03-16T00:27:43.246+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nhe.py for tasks to queue
[2023-03-16T00:27:43.247+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:27:43.247+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/nhe.py
[2023-03-16T00:27:43.254+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:27:43.252+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/nhe.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nhe.py", line 37
    if(qtd > 1000):
                  ^
TabError: inconsistent use of tabs and spaces in indentation
[2023-03-16T00:27:43.255+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nhe.py
[2023-03-16T00:27:43.275+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nhe.py took 0.036 seconds
[2023-03-16T00:28:14.036+0000] {processor.py:153} INFO - Started process (PID=2429) to work on /opt/airflow/dags/nhe.py
[2023-03-16T00:28:14.039+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nhe.py for tasks to queue
[2023-03-16T00:28:14.041+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:28:14.041+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/nhe.py
[2023-03-16T00:28:14.049+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:28:14.047+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/nhe.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nhe.py", line 37
    if(qtd > 1000):
                  ^
TabError: inconsistent use of tabs and spaces in indentation
[2023-03-16T00:28:14.051+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nhe.py
[2023-03-16T00:28:14.082+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nhe.py took 0.054 seconds
[2023-03-16T00:28:44.923+0000] {processor.py:153} INFO - Started process (PID=2479) to work on /opt/airflow/dags/nhe.py
[2023-03-16T00:28:44.927+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nhe.py for tasks to queue
[2023-03-16T00:28:44.928+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:28:44.928+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/nhe.py
[2023-03-16T00:28:44.941+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:28:44.939+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/nhe.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nhe.py", line 37
    if(qtd > 1000):
                  ^
TabError: inconsistent use of tabs and spaces in indentation
[2023-03-16T00:28:44.942+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nhe.py
[2023-03-16T00:28:44.969+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nhe.py took 0.056 seconds
[2023-03-16T00:29:15.934+0000] {processor.py:153} INFO - Started process (PID=2529) to work on /opt/airflow/dags/nhe.py
[2023-03-16T00:29:15.936+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nhe.py for tasks to queue
[2023-03-16T00:29:15.941+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:29:15.941+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/nhe.py
[2023-03-16T00:29:15.975+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:29:15.966+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/nhe.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nhe.py", line 37
    if(qtd > 1000):
                  ^
TabError: inconsistent use of tabs and spaces in indentation
[2023-03-16T00:29:15.978+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nhe.py
[2023-03-16T00:29:16.048+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nhe.py took 0.122 seconds
[2023-03-16T00:29:46.761+0000] {processor.py:153} INFO - Started process (PID=2579) to work on /opt/airflow/dags/nhe.py
[2023-03-16T00:29:46.764+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nhe.py for tasks to queue
[2023-03-16T00:29:46.765+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:29:46.765+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/nhe.py
[2023-03-16T00:29:46.776+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:29:46.774+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/nhe.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nhe.py", line 37
    if(qtd > 1000):
                  ^
TabError: inconsistent use of tabs and spaces in indentation
[2023-03-16T00:29:46.776+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nhe.py
[2023-03-16T00:29:46.798+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nhe.py took 0.043 seconds
[2023-03-16T00:30:17.389+0000] {processor.py:153} INFO - Started process (PID=2629) to work on /opt/airflow/dags/nhe.py
[2023-03-16T00:30:17.392+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nhe.py for tasks to queue
[2023-03-16T00:30:17.393+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:30:17.393+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/nhe.py
[2023-03-16T00:30:17.400+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:30:17.399+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/nhe.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nhe.py", line 37
    if(qtd > 1000):
                  ^
TabError: inconsistent use of tabs and spaces in indentation
[2023-03-16T00:30:17.401+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nhe.py
[2023-03-16T00:30:17.423+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nhe.py took 0.040 seconds
[2023-03-16T00:30:48.068+0000] {processor.py:153} INFO - Started process (PID=2680) to work on /opt/airflow/dags/nhe.py
[2023-03-16T00:30:48.071+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nhe.py for tasks to queue
[2023-03-16T00:30:48.073+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:30:48.072+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/nhe.py
[2023-03-16T00:30:48.080+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:30:48.078+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/nhe.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nhe.py", line 37
    if(qtd > 1000):
                  ^
TabError: inconsistent use of tabs and spaces in indentation
[2023-03-16T00:30:48.081+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nhe.py
[2023-03-16T00:30:48.103+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nhe.py took 0.043 seconds
[2023-03-16T00:31:18.203+0000] {processor.py:153} INFO - Started process (PID=2730) to work on /opt/airflow/dags/nhe.py
[2023-03-16T00:31:18.212+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nhe.py for tasks to queue
[2023-03-16T00:31:18.215+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:31:18.214+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/nhe.py
[2023-03-16T00:31:18.228+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:31:18.226+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/nhe.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nhe.py", line 37
    if(qtd > 1000):
                  ^
TabError: inconsistent use of tabs and spaces in indentation
[2023-03-16T00:31:18.229+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nhe.py
[2023-03-16T00:31:18.253+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nhe.py took 0.057 seconds
[2023-03-16T00:31:48.454+0000] {processor.py:153} INFO - Started process (PID=2778) to work on /opt/airflow/dags/nhe.py
[2023-03-16T00:31:48.455+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nhe.py for tasks to queue
[2023-03-16T00:31:48.456+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:31:48.456+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/nhe.py
[2023-03-16T00:31:48.460+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:31:48.459+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/nhe.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nhe.py", line 37
    if(qtd > 1000):
                  ^
TabError: inconsistent use of tabs and spaces in indentation
[2023-03-16T00:31:48.461+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nhe.py
[2023-03-16T00:31:48.480+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nhe.py took 0.029 seconds
[2023-03-16T00:32:18.641+0000] {processor.py:153} INFO - Started process (PID=2828) to work on /opt/airflow/dags/nhe.py
[2023-03-16T00:32:18.642+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nhe.py for tasks to queue
[2023-03-16T00:32:18.643+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:32:18.643+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/nhe.py
[2023-03-16T00:32:18.650+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:32:18.648+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/nhe.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nhe.py", line 37
    if(qtd > 1000):
                  ^
TabError: inconsistent use of tabs and spaces in indentation
[2023-03-16T00:32:18.652+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nhe.py
[2023-03-16T00:32:18.672+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nhe.py took 0.034 seconds
[2023-03-16T00:32:49.404+0000] {processor.py:153} INFO - Started process (PID=2876) to work on /opt/airflow/dags/nhe.py
[2023-03-16T00:32:49.406+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nhe.py for tasks to queue
[2023-03-16T00:32:49.407+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:32:49.407+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/nhe.py
[2023-03-16T00:32:49.412+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:32:49.411+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/nhe.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nhe.py", line 37
    if(qtd > 1000):
                  ^
TabError: inconsistent use of tabs and spaces in indentation
[2023-03-16T00:32:49.413+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nhe.py
[2023-03-16T00:32:49.430+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nhe.py took 0.028 seconds
[2023-03-16T00:33:19.629+0000] {processor.py:153} INFO - Started process (PID=2924) to work on /opt/airflow/dags/nhe.py
[2023-03-16T00:33:19.632+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nhe.py for tasks to queue
[2023-03-16T00:33:19.633+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:33:19.633+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/nhe.py
[2023-03-16T00:33:19.637+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:33:19.636+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/nhe.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nhe.py", line 37
    if(qtd > 1000):
                  ^
TabError: inconsistent use of tabs and spaces in indentation
[2023-03-16T00:33:19.638+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nhe.py
[2023-03-16T00:33:19.653+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nhe.py took 0.027 seconds
[2023-03-16T00:33:50.006+0000] {processor.py:153} INFO - Started process (PID=2965) to work on /opt/airflow/dags/nhe.py
[2023-03-16T00:33:50.011+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nhe.py for tasks to queue
[2023-03-16T00:33:50.013+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:33:50.013+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/nhe.py
[2023-03-16T00:33:50.019+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:33:50.018+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/nhe.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nhe.py", line 37
    if(qtd > 1000):
                  ^
TabError: inconsistent use of tabs and spaces in indentation
[2023-03-16T00:33:50.020+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nhe.py
[2023-03-16T00:33:50.038+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nhe.py took 0.036 seconds
[2023-03-16T00:34:20.651+0000] {processor.py:153} INFO - Started process (PID=3012) to work on /opt/airflow/dags/nhe.py
[2023-03-16T00:34:20.653+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nhe.py for tasks to queue
[2023-03-16T00:34:20.654+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:34:20.653+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/nhe.py
[2023-03-16T00:34:20.658+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:34:20.657+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/nhe.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nhe.py", line 37
    if(qtd > 1000):
                  ^
TabError: inconsistent use of tabs and spaces in indentation
[2023-03-16T00:34:20.658+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nhe.py
[2023-03-16T00:34:20.675+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nhe.py took 0.027 seconds
[2023-03-16T00:34:51.191+0000] {processor.py:153} INFO - Started process (PID=3060) to work on /opt/airflow/dags/nhe.py
[2023-03-16T00:34:51.194+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nhe.py for tasks to queue
[2023-03-16T00:34:51.195+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:34:51.194+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/nhe.py
[2023-03-16T00:34:51.200+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:34:51.199+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/nhe.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nhe.py", line 37
    if(qtd > 1000):
                  ^
TabError: inconsistent use of tabs and spaces in indentation
[2023-03-16T00:34:51.204+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nhe.py
[2023-03-16T00:34:51.224+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nhe.py took 0.035 seconds
[2023-03-16T00:35:21.266+0000] {processor.py:153} INFO - Started process (PID=3108) to work on /opt/airflow/dags/nhe.py
[2023-03-16T00:35:21.268+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nhe.py for tasks to queue
[2023-03-16T00:35:21.269+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:35:21.268+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/nhe.py
[2023-03-16T00:35:21.273+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:35:21.272+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/nhe.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nhe.py", line 37
    if(qtd > 1000):
                  ^
TabError: inconsistent use of tabs and spaces in indentation
[2023-03-16T00:35:21.274+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nhe.py
[2023-03-16T00:35:21.294+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nhe.py took 0.032 seconds
[2023-03-16T00:35:51.439+0000] {processor.py:153} INFO - Started process (PID=3156) to work on /opt/airflow/dags/nhe.py
[2023-03-16T00:35:51.440+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nhe.py for tasks to queue
[2023-03-16T00:35:51.441+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:35:51.441+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/nhe.py
[2023-03-16T00:35:51.445+0000] {logging_mixin.py:137} INFO - [2023-03-16T00:35:51.444+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/nhe.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nhe.py", line 37
    if(qtd > 1000):
                  ^
TabError: inconsistent use of tabs and spaces in indentation
[2023-03-16T00:35:51.445+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nhe.py
[2023-03-16T00:35:51.462+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nhe.py took 0.026 seconds
